website/docs/api/index.md

```diff
@@ -22,10 +22,11 @@ The internal API used by the Admin UI (unleash-frontend). Since v4.0.0 all opera
 
 With versions earlier than v4.0.0 and `insecure` authentication Basic Auth (with curl `-u myemail@test.com:`) is enough.
 
-- [Feature Toggles API](/api/admin/features)
-- [Strategies API](/api/admin/strategies)
 - [Events API](/api/admin/events)
+- [Feature Toggles API](/api/admin/features)
 - [Metrics API](/api/admin/metrics)
+- [Project API](/api/admin/project.md)
+- [Strategies API](/api/admin/strategies)
```

Comment:
These have been sorted and I've added the Projects API link.

Reply:
"Project API" is the only one with `.md`, though not sure if it matters.